recipe/meta.yaml

```diff
@@ -47,7 +47,6 @@ about:
     designed to facilitate packaging Python projects.
   doc_url: http://pythonhosted.org/setuptools/
   dev_url: https://github.com/pypa/setuptools
-  dev_source_url: https://github.com/pypa/setuptools/tree/master/docs
```

Comment:
Can we fix the linter instead.  We use this key in AnacondaRecipes.

Reply:
Oops, this should be `doc_source_url` which the linter does understand

Reply:
Ah ok. Thanks for the clarification. Thought `dev_source_url` sounded weird. Also thanks for correcting it. 😄